db/primary_migrate/20241015154109_create_socure_reason_codes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSocureReasonCodes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :socure_reason_codes do |t|
+      t.string :code, comment: 'sensitive=false'
+      t.string :group, comment: 'sensitive=false'
+      t.text :description, comment: 'sensitive=false'
+      t.datetime :added_at, comment: 'sensitive=false'
+      t.datetime :deactivated_at, comment: 'sensitive=false'
+
+      t.timestamps comment: 'sensitive=false'
+
+      t.index :code, unique: true
```

Comment:
Do you think there is any benefit to adding an index on `deactivated_at`, and maybe adding a scope to the model, so we can do like `SocureReasonCodes.active`? I feel like we're probably going to almost always filter out anything that's been deactivated.

(Or maybe that's a problem that will come when we start using these in code.)

Reply:
Yeah, I think a deactivated_at index may be helpful. Let me look at some of these queries. I don't want to add scopes we are not using.